homework/test/test-list.js

```diff
@@ -3,6 +3,10 @@ describe('Test > ', function() {
   });
 
   it('will test some pure functions', function() {
-    // Write any pure function assertion here.
+   var a = 5;
```

Comment:
Write it as a method then call it as the test target, instead of write the expressions here.